src/resourceClient/resourceClient.ts

```diff
@@ -43,11 +43,11 @@ export class ResourceClient<T extends ResponseRoot> extends BaseResourceClient {
 
   async get(id: string, options?: Options): Promise<T> {
     const url = this.urlBuilder.getOneUrl(id, this.getStorefront(options?.storefront), options);
-    return this.getByUrl(url, options);
+    return await this.getByUrl(url, options);
```

Comment:
there is no reason to add additional await, the method will return promise, so types should match

Reply:
It's part of the eslint rules we are using in all of our projects, await is required